### ISSUE TITLE
Email: IBM Comments on WCAG 3.0 Draft (6 - Terminology & Naming)

### ISSUE THREAD

**Issue by cwadamsoforacle** [w3c/silver#500]
Functional categories
We like the use of functional categories to drive the rationale behind the guidelines. We have a few comments on the functional categories:
•	Functional categories defined in WCAG 3 do not have equivalent levels of granularity. For example, Vision and Visual is one functional category, which encompasses a number of subcategories which may have non-aligned and even have opposing needs.  Even ignoring those, the needs of blind and low vision users are significantly different, both in modes of operation and in functional needs. Low Vision alone can encompass users with such disparate needs as visual acuity, light sensitivity, contrast sensitivity, shape perception, field of vision, color vision and motion tracking. It seems likely that the diversity of needs within the vision category is greater than that of other categories, as it relates to author considerations, yet it is encompassed by a single functional category in WCAG 3. As a second example, the Cognitive category is split into 7 sub-categories in WCAG 3, while 508 only has one combined category for 302.9 With Limited Language, Cognitive, and Learning Abilities, similar to the one category for EN 301 549’s 4.2.10 Usage with limited cognition, language or learning. Suggest at the minimum to adopt a harmonized list of functional categories similar to the Revised 508 Standards Functional Performance Criteria and EN 301 549 Functional Performance Statements.
•	References to the European EN 301 549 standard should be to the latest V3.1.1, not the earlier V1.1.1, freely available from the ETSI.org website.
•	It is unclear if photosensitivity and vestibular triggers are included under Vision and Visual or maybe under one of the intersection categories.
•	Not sure if working on a separate functional needs document is necessary when the ISO/IEC 29138-1:2018 document contains an extensive list already, granted it's behind a paywall. Addressing functional needs is important, and these can be mapped to the functional categories documented for the guideline. The ISO standard used to have a mapping of user needs to functional categories, but the latest update eliminated that mapping.
•	Wonder if functional categories should be documented on the individual outcomes (or how-to's) vs. in the guideline. 
•	In the list of functional needs in the Cognitive category: Attention sub-section, the “Use with limited ability to direct attention” seems synonymous with “Use with limited ability to focus attention”. Most people aren’t experts on cognitive accessibility, so they’ll need some explanation of the differences between these two. Otherwise, consider simplifying the two items to a single functional need. 
[IBM comments on the WCAG 3 FPWD -March 2.docx](https://github.com/w3c/silver/files/6174003/IBM.comments.on.the.WCAG.3.FPWD.-March.2.docx)

**Comment by cwadamsoforacle** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.
